ember.js/2019-10/Meeting_Ember_Core_Team_20191004.md

```diff
@@ -0,0 +1,108 @@
+# Meeting: Ember Core Team - Oct 4, 2019
+
+## Agenda
+
+- Determine note taker (JW)
+- Determine person to upload notes (JW)
+- Octane links
+- Octane blueprints location
+- Including ember-auto-import
+- jQuery migrations
+
+## Attendees
+
+- [x]  Yehuda Katz
+- [x]  Tom Dale
+- [x]  Kris Selden
+- [ ]  Leah Silber
+- [x]  Robert Jackson
+- [x]  Igor Terzic
+- [ ]  Matthew Beale
+- [x]  Edward Faulkner
+
+- [x]  Dan Gebhardt
+- [x]  Godfrey Chan
+- [ ]  Ricardo Mendes
+- [x]  Chad Hietala
+- [ ]  Katie Gengler
+- [x]  Melanie Sumner
+- [x]  Jen Weber
+- [x]  Chris Garrett
+
+## Minutes
+
+### Octane links
+
+RJ: We should do a recap of the Octane Strike Team Meetings, since they cover a lot of what we discuss here
+
+JW: To summarize, we'll keep linking to the edition page when we can, and will not go back and change blog URLs. A few weeks after Octane ships, we'll make a plan for smooth redirects etc. for the existing Guides preview pages
+
+YK: There is a postmortem to do but not yet
+
+JW: agreed
+
+YK: We still need to update the wording around the links of the blog post too
+
+JW: Yes, will do that now
+
+### Octane Blueprints location
+
+RJ: There are two main options, we can go with standalone package as blueprints, or have ember new without a flag do whatever is in Ember CLI. Some community members seemed surprised by keeping the app blueprint and the addon blueprint in CLI vs doing what we show in the guides now. I strongly prefer to keep blueprints in the cli package, and think an RFC would be good to make blueprints into their own package.
```

Comment:
fwiw, I was suggesting that `ember new my-app` _implicitly_ uses the external octane blueprint. ;)

Reply:
@NullVoxPopuli - Yep, I definitely understood you there.